### PR TITLE
added materialized-views workloadgoup description

### PR DESCRIPTION
**Pull request by ariela-boursi**

### CODE REVIEW

**Comment by shsagir** [comment]
Learn Build status updates of commit _[6a022d9](https://github.com/ariela-b/dataexplorer-docs/commits/6a022d9b7451f6553411788722244d42ef46d3d1)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[data-explorer/kusto/management/materialized-views/materialized-view-overview.md](https://github.com/ariela-b/dataexplorer-docs/blob/uaer/arielab/MaterializedView-WorkloadGroup/data-explorer/kusto/management/materialized-views/materialized-view-overview.md) | :warning:Warning | | [Details](#user-content-44dd904a259928027f5786bed1c3b862-582443c6)
[data-explorer/kusto/management/workload-groups.md](https://github.com/ariela-b/dataexplorer-docs/blob/uaer/arielab/MaterializedView-WorkloadGroup/data-explorer/kusto/management/workload-groups.md) | :white_check_mark:Succeeded | |

<a id="44dd904a259928027f5786bed1c3b862-582443c6"></a>
### [data-explorer/kusto/management/materialized-views/materialized-view-overview.md](https://github.com/ariela-b/dataexplorer-docs/blob/uaer/arielab/MaterializedView-WorkloadGroup/data-explorer/kusto/management/materialized-views/materialized-view-overview.md)
 - **Line 181, Column 44**: **[Warning: bookmark-not-found]** `````Cannot find bookmark '#materializedviews-workload-group' in 'kusto/management/workload-groups.md', did you mean '#materialized-views-workload-group'?`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/44fbb50c-a02d-79e7-9b6a-0040a309a8dd/202212291625098723-1554/BuildReport?accessString=bb5f96c0924a94134e72868477a5be9c6977651fe4c5966ea57826cb55ec9e1c).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.learn.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Learn Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/44fbb50c-a02d-79e7-9b6a-0040a309a8dd?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings or suggestions unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.learn.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Comment by ariela-boursi** [comment]
#assign-reviewer:yifatsor

**Comment by shsagir** [comment]
Learn Build status updates of commit _[f2d6780](https://github.com/ariela-b/dataexplorer-docs/commits/f2d67801624b9fa45eca04d60c38957d970cec21)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[data-explorer/kusto/management/materialized-views/materialized-view-overview.md](https://github.com/ariela-b/dataexplorer-docs/blob/uaer/arielab/MaterializedView-WorkloadGroup/data-explorer/kusto/management/materialized-views/materialized-view-overview.md) | :white_check_mark:Succeeded | |
[data-explorer/kusto/management/workload-groups.md](https://github.com/ariela-b/dataexplorer-docs/blob/uaer/arielab/MaterializedView-WorkloadGroup/data-explorer/kusto/management/workload-groups.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/44fbb50c-a02d-79e7-9b6a-0040a309a8dd/202212291630499504-1554/BuildReport?accessString=5f2f086caacd5a15dfdaff31dc3f185b99aed7bae627d4be0b3e6ac49ac81dd5).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/44fbb50c-a02d-79e7-9b6a-0040a309a8dd?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Comment by jborsecnik** [comment]
@shsagir, 

Can you review the proposed changes? 

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: data-explorer/kusto/management/workload-groups.md [hunk 1]
```diff
@@ -52,6 +52,22 @@ You can't:
 
 You can [monitor](#monitoring) what gets classified to the `internal` workload group, and statistics of those requests.
 
+
+### Materialized-Views workload group
+
+The `materialized-views` workload group is populated with [materialized-views](materialized-views/materialized-view-overview.md) requests only.
```
Comment (yifatsor) [inline data-explorer/kusto/management/workload-groups.md:58]: This is confusing since some might think it applies to queries over MVs as well. I would rephrase to something like: 
"The `$materialized views` workload group applies to materialized views materialization process (see how materialized views work)." - how materialized views work should be a link to the relevant section in the overview page

File: data-explorer/kusto/management/workload-groups.md [hunk 2]
```diff
@@ -9,14 +9,14 @@ ms.date: 04/30/2021
...
-Workload groups are defined at the cluster level. Up to 10 custom workload groups may be defined in addition to the two built-in workload groups.
+Workload groups are defined at the cluster level. Up to 10 custom workload groups may be defined in addition to the three built-in workload groups.
 
 > [!NOTE]
 > Requests that aren't queries or control commands aren't included in the scope of workload groups. For example: streaming ingestion requests.
 
 ## Built-in workload groups
 
-There are two pre-defined workload groups: [the `internal` workload group](#internal-workload-group) and [the `default` workload group](#default-workload-group).
+There are three pre-defined workload groups: [the `internal` workload group](#internal-workload-group), [the `default` workload group](#default-workload-group) and [the `materialized-views` workload group](#materialized-views-workload-group).
```
Comment (yifatsor) [inline data-explorer/kusto/management/workload-groups.md:19]: the name is actually `$materialized-views`, right?

File: data-explorer/kusto/management/materialized-views/materialized-view-overview.md [hunk 3]
```diff
@@ -177,6 +177,9 @@ The main contributors that can impact a materialized view health are:
 
 * **Materialized view definition**: The materialized view definition must be defined according to query best practices for best query performance. For more information, see [create command performance tips](materialized-view-create.md#performance-tips).
 
+* **Materialized view workload group limits**: By default materialized views are limited by the maximum amount of memory a query can allocate and by the percentage of nodes/threads on each node to fan out query execution to.
```
Comment (yifatsor) [inline data-explorer/kusto/management/materialized-views/materialized-view-overview.md:180]: I think this should be added as a sub-bullet under the *Cluster resources* bullet (first in this section). Consider something like: 
        `* By default, the [materialization process](link to how materialized views work) is limited by the amount of memory and CPU it can consume. These limits are defined, and can be changed, in the [materialized views workload group](link to the WG section).`

**Comment by yifatsor** [comment]
@ariela-boursi - I think it's worth mentioning the WG also under the monitoring section, specifically in the *Insufficient Resources*  result metric. We should say that if materialization hit memory limits, the WG can be modified, and it would be good to provide an example of an alter-merge command for the Max Memory Peak Per Node (to 64GB) - that's the most common property that customers will need to change. We should also mention that we don't recommend setting this value to more than 50% to the total amount of memory on the node.

**Comment by YaelSchuster** [comment]
Hi @ariela-boursi, I need you to please make this change on the private repo (dataexplorer-docs-**pr**) and not the public repo. 

Please reach out if you have any questions.

Thanks

#please-close

File: data-explorer/kusto/management/workload-groups.md (hunk 2 above)
Comment (ariela-boursi) [reply data-explorer/kusto/management/workload-groups.md:19]: yes
